scripts/verify-install.sh

```diff
@@ -35,6 +35,7 @@ KEYS+=("E85497D2DBA0EB9ADB0024279BCD95C4FF296868 yyforyongyu")
 KEYS+=("32F7EA1E7A0339F7D37164B9F82D456EA023C9BF hieblmi")
 KEYS+=("5295A477FFC8064D7057B191FA7E65C951F12439 proofofkeags")
 KEYS+=("3E9BD4436C288039CA827A9200C9E2BC2E45666F suheb")
+KEYS+=("5F75437E11695F86D50C11BB1AFF9C4DCED6D666 ziggie")
```

Comment:
Yeah, good catch. It's not a problem, it can be different. You just have to remember to call your signature file the correct way each time you sign a release. It needs to match the pattern `manifest-<name>-v0.xx.yy-beta.sig`, where `<name>` is what's in the `verify-install.sh` file, otherwise the signature won't be counted. If you're okay with that not matching your GitHub username, that's fine.

Reply:
cool yes I am fine with this, thank you for pointing this out.

Reply:
updated to the github username.